examples/js/controls/OrbitControls.js

```diff
@@ -151,9 +154,12 @@ THREE.OrbitControls = function ( object, domElement ) {
 			}
 
 			if ( scope.enableDamping ) {
+				var deltaTime = clock.getDelta();
+				_dampingFactor = scope.dampingFactor * deltaTime * 60;
+				_dampingFactor = Math.min(_dampingFactor, 1);
```

Comment:
Not sure I understand why `Math.min()` is needed here 🤔

Reply:
I found a existing same logic in [this example](https://github.com/mrdoob/three.js/blob/dev/examples/webgl_gpgpu_birds_gltf.html#L645) .

Reply:
Like I said before, it's not good that such logic is spread over the code base, see https://github.com/mrdoob/three.js/pull/20638#issuecomment-723480061.

Reply:
[Here is a good article on framerate independent damping](http://www.rorydriscoll.com/2016/03/07/frame-rate-independent-damping-using-lerp/). With taht in mind, I find this formula works quite well:
```javascript
const _dampingFactor = Math.pow( 1 - scope.dampingFactor, deltaTime * 60 );
```

Reply:
@arodic Thanks! I'll learn it.

Reply:
I found it's hard to use this formula with deltaTime, so I changed to elapsedTime, resulting [this version](https://raw.githack.com/gonnavis/three.js/OrbitControlsDampingSpeedConsistentElapsedTime/examples/misc_controls_orbit.html), [compare](https://github.com/mrdoob/three.js/compare/dev...gonnavis:OrbitControlsDampingSpeedConsistentElapsedTime).
But with this change, may need large refactoring of code, and now just damping the rotate not yet pan, and has a strange initial camera position problem to be solve, so I think I'll done it later.
BTW, because of this formula goes infinitely near zero, so should don't need care about tab changing problem above metioned.